src/config/crd/bases/goharbor.goharbor.io_assessmentreports.yaml

```diff
@@ -128,6 +128,23 @@ spec:
                       elasticSearchUser:
                         description: ElasticSearch username for the client
                         type: string
+                      governor:
```

Comment:
Why do we need the governor configuration on each assessment report? That looks strange as there are going to be many of those.